metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -30,6 +23,17 @@ class DatahubConfig(BaseModel):
     gms: GmsConfig
 
 
+def write_datahub_config(host: str, token: Optional[str]):
+    config = {
+        "gms": {
+            "server": host,
+            "token": token,
```

Comment:
will this write "token": None if token is not provided?

Reply:
@shirshanka If token is None, then no Authorization header will be attached